source/MaterialXCore/Document.cpp

```diff
@@ -7,8 +7,10 @@
 
 #include <MaterialXCore/Util.h>
 #include <MaterialXCore/Version.h>
+#include <MaterialXCore/Node.h>
```

Comment:
It looks like these two added headers (Node.h and unordered_set) are no longer needed?

Reply:
Could be. I'll do a clean up here and other places you've pointed out.